src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorHyperLinkDialog.java

```diff
@@ -83,7 +83,7 @@ private void indexLinkCategories() {
           throw new NoSuchElementException("Non-existing link category selected");
       }
     }
-    PageObjectLogging.log("viewResults", "Category indexes sorted", true);
+    Log.log("viewResults", "Category indexes sorted", true);
   }
 
   private boolean isCategoryResult(int category) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "isCategoryResult" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)